### PR TITLE
product listing page added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR product listing page is added, due to some problem, it is not visible on the preview. 
Please review the code and help me for improvement

### CODE REVIEW

**Review by munjalSaksham71 — APPROVED** [review]
Likes that you have breakdown all the components. Also the Code is well structured

File: src/components/filterCategory/filterPrice/filterPrice.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import '../filterPrice/filterPrice.css';
+
+export const FilterPrice = () => {
+    const { dispatch, state } = useFilter();
+    const { priceValue } = state;
+    return (
+        <div className="filter--price">
+            <p className="filter__heading">Price</p>
+            <input type="range" min="50000" max="160000" className="slider"
+                onChange={(e) => dispatch({ type: "PRICE_RANGE", priceValue: e.target.value })} id="filterRange" name='range' />
```
Comment (Raghav888) [inline src/components/filterCategory/filterPrice/filterPrice.jsx:10]: you have created small components, thats great

File: src/components/filterCategory/filterRating/filterRating.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <p className="filter__heading">Rating</p>
+            <input type="radio" name="rating" id="four-and-above" defaultValue="four-and-above" onClick={() => dispatch({ type: "RATING", payload: "FOUR_AND_ABOVE" })} />
+            <label htmlFor="four-and-above">4 stars and above</label>
+            <br />
+            <input type="radio" name="rating" id="three-star" defaultValue="three-star" onClick={() => dispatch({ type: "RATING", payload: "THREE_STAR" })} />
+            <label htmlFor="three-star">3 stars</label>
+            <br />
+            <input type="radio" name="rating" id="two-star" defaultValue="two-star" onClick={() => dispatch({ type: "RATING", payload: "TWO_STAR" })} />
+            <label htmlFor="two-star">2 stars</label>
+            <br />
```
Comment (Raghav888) [inline src/components/filterCategory/filterRating/filterRating.jsx:16]: you can create an array of  object with data and map over it

**Review by Raghav888 — APPROVED** [review]

File: src/components/filterCategory/filterPrice/filterPrice.jsx (hunk 1 above)
Comment (ShreyashMahajan) [reply src/components/filterCategory/filterPrice/filterPrice.jsx:10]: ty

File: src/components/filterCategory/filterRating/filterRating.jsx (hunk 2 above)
Comment (ShreyashMahajan) [reply src/components/filterCategory/filterRating/filterRating.jsx:16]: ok thank you, i will look upon that

**Comment by ShreyashMahajan** [comment]
> Likes that you have breakdown all the components. Also the Code is well structured

thank you

**Pull request merged**